features/collections/collection_reveals_notifications.feature

```diff
@@ -0,0 +1,296 @@
+@collections @promptmemes @giftexchanges @challenges @works @gifts
+
+Feature: Notification emails for newly revealed collections
+  When a moderator reveals or de-anons an entire collection, the relevant notification emails should be sent.
+
+  #########################################
+  #
+  # WORKS POSTED DIRECTLY TO THE COLLECTION
```

Comment:
For works posted to a moderated+unrevealed collection, tests cover: gift/related/prompt fill x unapproved/approved/rejected = 9. Very thorough!

Reply:
I did notice some duplication between these tests and the ones in collection_anonymity.feature, but I'm going to fix that in the PR for AO3-2240, which should be coming up later tonight.

Reply:
Do we want this one merged first? So you don't have to fight cascading conflicts.

Reply:
That would be great, yeah.